### PR TITLE
GitHub actions

### PR DESCRIPTION
**Pull request by marcserrat**
Moved from Travis CI to GitHub Actions
Publishing on PyPI happens on taging

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        pip install -r requirements/dev.txt
+        pip install -r requirements/test.txt
+        pip install pytest-cov
```
Comment (vgrebenschikov) [inline .github/workflows/build.yml:27]: why not in test.txt?

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      uses: codecov/codecov-action@v1
+      with:
+        file: ./tests/reports/coverage.xml
+        fail_ci_if_error: true
+        verbose: true
+    - name: SonarCloud
+      uses: SonarSource/sonarcloud-github-action@master
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
```
Comment (vgrebenschikov) [inline .github/workflows/build.yml:44]: newline?

File: .github/workflows/deploy.yml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+    - name: Publish to PyPI
+      env:
+        TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
+        TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
+      run: |
+        python setup.py sdis bdist_wheel
+        twine upload dist/*
```
Comment (vgrebenschikov) [inline .github/workflows/deploy.yml:49]: newline?

File: .github/workflows/build.yml (hunk 1 above)
Comment (marcserrat) [reply .github/workflows/build.yml:27]: ok, was duplicated on travis. I remove

File: .github/workflows/build.yml (hunk 2 above)
Comment (marcserrat) [reply .github/workflows/build.yml:44]: OK

File: .github/workflows/deploy.yml (hunk 3 above)
Comment (marcserrat) [reply .github/workflows/deploy.yml:49]: ok

File: .github/workflows/deploy.yml [hunk 4]
```diff
@@ -0,0 +1,48 @@
+name: Build Django-RQL library
```
Comment (ffaraoneim) [inline .github/workflows/deploy.yml:1]: Name should be: Deploy Django-RQL library

**Review by ffaraoneim — CHANGES_REQUESTED** [review]

File: .github/workflows/deploy.yml (hunk 4 above)
Comment (marcserrat) [reply .github/workflows/deploy.yml]: Awesome! Updated

**Review by ffaraoneim — APPROVED** [review]

File: .github/workflows/deploy.yml [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+      uses: SonarSource/sonarcloud-github-action@master
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+    - name: Publish to PyPI
+      env:
+        TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
+        TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
+      run: |
+        python setup.py sdis bdist_wheel
```
Comment (ffaraoneim) [inline .github/workflows/deploy.yml:47]: typo: should be python setup.py **sdist** bdist_wheel

**Review by ffaraoneim — CHANGES_REQUESTED** [review]

**Review by ffaraoneim — APPROVED** [review]

**Review by vgrebenschikov — APPROVED** [review]

**Pull request merged**